template-manager-cmtemplate/src/main/resources/definitions/cm-cloud-storage-location-specification.json

```diff
@@ -0,0 +1,64 @@
+{
+  "entries": [
+    {
+      "propertyName": "zeppelin.notebook.dir",
+      "propertyFile": "zeppelin-site",
+      "description": "The directory where Zeppelin notebooks are saved",
+      "defaultPath": "{{{ defaultPath }}}/zeppelin/notebook",
+      "propertyDisplayName": "Zeppelin Notebooks Root Directory",
+      "relatedServices": ["ZEPPELIN_SERVER"],
+      "requiredForAttachedCluster": false,
+      "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
+    },
+    {
```

Comment:
Zeppelin requires different configs for s3 and non-s3. For s3 it requires zeppelin.notebook.s3.bucket and for other file system it requires zeppelin.notebook.dir... can this be supported ?

Reply:
I see, the supported storage is used for this. I am going to remove everything else for zeppelin, just keep the S3

Reply:
Fixed as requested